### DISCUSSION TITLE
How to switch theme in reflex 0.4.3

### DISCUSSION THREAD

**Discussion by amanchaudhary-95** [reflex-dev/reflex#2820] [Q&A] [answered]
In reflex 0.3.10 i.e. on chakra UI, The theme was switchable via simply rx.color_mode_button(rx.color_mode_icon()). It used to give a theme switch button. I can't find such a function in the new Redix UI-based reflex.
Currently, I can set the theme for the entire app via rx.App(theme = rx.theme(appearance="light"....)) but I can't make a button to switch the theme. The closest thing is rx.theme_panel(). But it's not the same as the previous one. How to produce the same theme switch button in new reflex 0.4.3?
![image](https://github.com/reflex-dev/reflex/assets/94781328/6cccde6e-e496-404e-bf9f-b7555296949d)

**Comment by picklelo** [comment, marked as answer]
Looks like we missed the documentation for this.

You can use 
```python
rx.color_mode.button("Dark Mode"),
```
To achieve this now. We will update our website with this info
